element.go

```diff
@@ -472,7 +482,9 @@ func (el *Element) ResourceE() ([]byte, error) {
 	var bin []byte
 	if res.Base64Encoded {
 		bin, err = base64.StdEncoding.DecodeString(data)
-		utils.E(err)
+		if err != nil {
```

Comment:
This error is impossible, the value is produced by chrome protocol, if the data format is corrupt, it will be a fatal bug of chrome.